baseline/client/tests/verbal-fluency.test.js

```diff
@@ -2,19 +2,17 @@ require("@adp-psych/jspsych/jspsych.js");
 import { VerbalFluency } from "../verbal-fluency/verbal-fluency.js";
 import { pressKey } from "./utils.js";
 
-jest.useFakeTimers('legacy'); // why legacy: https://github.com/facebook/jest/issues/11500
-
-beforeEach(() => {
```

Comment:
Removing this beforeEach block has made the tests brittle. The "records letter prompt and user response" test now only works because the test before it happens to call jest.useFakeTimers. If that test gets moved, skipped, or changed to not call useFakeTimers then the "records letter prompt and user response" test will no longer work. Please add a beforeEach that calls jest.useFakeTimers and an afterEach that calls jest.useRealTimers. (Which will also allow you to DRY things up by pulling all of the jest.useFakeTimers calls out of the individual tests, of course.)

Reply:
Right, that makes perfect sense!